tests/ci/cdk/cdk/codebuild/github_ci_linux_arm_omnibus.yaml

```diff
@@ -148,6 +148,14 @@ batch:
         compute-type: BUILD_GENERAL1_LARGE
         image: ECR_REPO_PLACEHOLDER:ubuntu-20.04_clang-10x_latest
 
+    - identifier: ubuntu2004_clang10x_aarch_prefix
+      buildspec: ./tests/ci/codebuild/linux-aarch/run_prefix_tests.yml
```

Comment:
Nit: maybe add a comment with a rust pkg/repo link(which this dimension try to protect)?

Reply:
I don't think Rust is directly relevant --  we want to protect the prefix build regardless of Rust.  

Also, the Rust build utilizing the prefix build is still just a proof-of-concept, so there isn't really any link to provide that would be viable long-term.